src/main/java/edu/harvard/iq/dataverse/flyway/StartupFlywayMigrator.java

```diff
@@ -0,0 +1,28 @@
+package edu.harvard.iq.dataverse.flyway;
+
+import org.flywaydb.core.Flyway;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.sql.DataSource;
+
+@Startup
+@Singleton
```

Comment:
While reading into Flyway in a Java EE context, I [stumbled over the annotation](https://web.archive.org/web/20180825121637/http://are-you-ready.de/blog/2017/06/08/integrating-flyway-with-java-ee-and-using-its-datasource)
`@TransactionManagement(value = TransactionManagementType.BEAN)`.

The explanation sounds reasonable - should this be added here?